FRENCH/src/ch20-01-single-threaded.md

```diff
@@ -142,7 +142,7 @@ will return a new `TcpListener` instance. The reason the function is called
 La fonction `bind` dans ce scénario fonctionne comme la fonction `new` dans le
 sens où elle retourne une nouvelle instance de `TcpListener`. La raison pour
 laquelle cette fonction s'appelle `bind` *(NdT : signifie "lier")* est que dans
-le domaine des réseaux, se connecter un port à écouter se dit se “lier à un port”.
+les réseaux, connecter un port à écouter se dit aussi “lier à un port”.
```

Comment:
```suggestion
le domaine des réseaux, se connecter un port à écouter se dit se “lier à un port”.
```

Reply:
my bad, "se connecter *à* un port"

Reply:
You can see in #249 how to do suggestions 😉

Are you commit all your changes from #359 in this Pull Request, or I do ?

Reply:
Could you do, please, because I'm currently working & I don't want do more mistakes before having time to read the doc you sent ...

Reply:
okay ! but please don't do this again, this is tedious 😁

Reply:
thanks, I won't, Sorry for the inconvenience...